.changeset/modern-keys-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudflare/next-on-pages': minor
+---
+
+Fix buildSuccess check that keeps users from receiving useful information.
```

Comment:
```suggestion
---
'@cloudflare/next-on-pages': patch
---

prevent buildSuccess check from crushing with a bad error message
```
this is fixing a bug so it should be a patch I think 🙂 

moreover the message can be improved (sorry I think you followed the initial title of the issue that I wrote but I got it a bit off, that's my bad)

Reply:
yeah, I just followed the initial title. No worries have added the change
<img width="542" alt="image" src="https://user-images.githubusercontent.com/62441572/231882558-05e319f8-5a08-415f-878b-1dbda8b4df7d.png">